news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+# Work on the new theme has started
+
+For a lot of time, the community asked us to rework the classic theme because many things were missing inside it. We announce that we are working on a new front office theme, and you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
+
+## Why such a decision?
+
+We received a lot of feedback from the community, with suggestions like:
+
+-   Upgrade to Bootstrap 5
+-   Rework the structure inside  `src`  directory, which was  `_dev`  before.
+-   Remove the `core.js` file. Theme developers can't modify anything inside it or even remove it from their theme.
+-   Use lazy loading.
+-   Choose, and use some CSS class naming strategy consistently, like BEM. This is more maintainable and easier to read.
+-   Use JSON-LD instead of microdata attributes.
+-   Add as many hooks as possible to extend the base theme.
+-   Separate the PrestaShop styling from the Bootstrap raw styling.
+-   Reduce the number of dependencies.
+-   Stop using jQuery in the theme. This would be the first step because modules use it anyway, so we can't remove it.
+-   Most of the time, JavaScript completely replaces some big parts of the content. Instead, it should update only a small part of it when needed.
+-   Add as many events as possible in the JavaScript part so modules and themes can use them.
+-   Use as many Bootstrap components as possible.
+
+It was pretty impossible to fix all these issues on the classic theme without breaking the whole ecosystem of modules and community themes. We decided that it was time for a change and a new theme. We can't change everything, but we can significantly improve the default PrestaShop theme.
+
+## How can I try it?
+
+Clone it from the theme repository, run `npm install`, `npm run build`, activate the theme and off you go!
+
+## Good base for theme developers
+
+In the new theme, we separated the CSS to two packs. First one is the functionality part, the second is purely for styling.
+
+Simply commenting out `@import "custom/_custom";` in theme.scss file will get you basically a pure Boostrap 5 theme base. You can get working immediately and not waste time by cleaning things you don't need.
+
+## When will it be released?
+
+Because of the complexity of this task, we decided to avoid writing about the release date. This means that depending on the time every people involved can work on it, the first version of the theme can be released in PrestaShop v8, v9, or later.
+
+We hope to release an Alpha version of the new theme at the same time as the v8 release, but it could be after, even inside a minor version.
```

Comment:
```suggestion
We hope to release an Alpha version of the new theme at the same time as the v8 release, but it could be after, even inside a minor version.
```

Need ref to v8
Are you talking about beta release or final release?

Reply:
Talking about Alpha

Reply:
Which ref are you talking about?

Reply:
> at the same time as the v8 release

If it's v8 beta it might be April
If it's v8 finale it might be summer